nemo/collections/asr/modules/conv_asr.py

```diff
@@ -27,20 +30,42 @@
     NeuralType,
     SpectrogramType,
 )
+from nemo.utils import logging
 from nemo.utils.decorators import experimental
 
 __all__ = ['ConvASRDecoder', 'ConvASREncoder', 'ConvASRDecoderClassification']
 
 
 @experimental
-class ConvASREncoder(NeuralModule):
+class ConvASREncoder(NeuralModule, Exportable):
     """
     Convolutional encoder for ASR models. With this class you can implement JasperNet and QuartzNet models.
     Based on these papers:
         https://arxiv.org/pdf/1904.03288.pdf
         https://arxiv.org/pdf/1910.10261.pdf
     """
 
+    def prepare_for_export(self) -> (Optional[torch.Tensor], Optional[torch.Tensor]):
```

Comment:
If prepare_for_export is in-place op (which I agree it should be) maybe we should use trailing underscore per Torch convention ?